src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -304,6 +304,11 @@ pub struct Args {
     )]
     orchestrator_process_data_directory: PathBuf,
 
+    /// The init container to use for computed and storaged when using the
+    /// kubernetes orchestrator.
+    #[clap(long)]
+    k8s_init_container_image: Option<String>,
```

Comment:
This should be called `orchestrator_kubernetes_init_container_image`, and have an env var named `ORCHESTRATOR_KUBERNETES_INIT_CONTAINER_IMAGE`, for consistency with the existing CLI flags.